helmfile/values/velero-sc.yaml.gotmpl

```diff
@@ -30,6 +30,8 @@ configuration:
   provider: gcp
   {{- end }}
 
+  defaultVolumesToRestic: true
+
```

Comment:
Not sure we need to do this change, but maybe it's better to just have this.

But with this you can remove this annotation https://github.com/elastisys/compliantkubernetes-apps/blob/6d40c7a135a6e2839d498dff9bc009ba61be3ade/helmfile/values/grafana-user.yaml.gotmpl#L100